docs/workforce-management-framework/upstream-1/introducing-features-and-modules.adoc

```diff
@@ -1,11 +1,25 @@
 include::topics/shared/attributes.adoc[]
 
-//':chapter:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
-:chapter: introducing-features-and-modules
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: introducing-features-and-modules
+
+= {WFM-RC-NameShort} Features
+
+include::topics/con-keycloak.adoc[leveloffset=+1]
+
+ifdef::downstream[]
+include::topics/con-logging.adoc[leveloffset=+1]
+endif::downstream[]
 
-= {WFM-RC-NameLong} Features
 include::topics/con-introducing-securityfeatures.adoc[leveloffset=+1]
 
-= {WFM-RC-NameLong} Modules
+== Introducing {WFM-RC-NameShort} Security Strategies
```

Comment:
I introduced the 'concepts' for both Security Strategies under features as I think we should be informing the reader at this point. Happy to take this out if you think it shouldn't be there.